.github/workflows/build.yml

```diff
@@ -224,6 +224,44 @@ jobs:
           message: |
             ⚠️ This PR is opened against **master**. You probably want to open it against **develop**.
 
+  job_external_contributor:
+    name: External Contributors
+    needs: job_install_deps
+    runs-on: ubuntu-20.04
+    if: |
+      github.event_name == 'pull_request'
+      && (github.action == 'opened' || github.action == 'reopened')
```

Comment:
do not run this on each update, as it is kind of wasteful (it should not actually error, just unnecessary computation)